src/userdel.c

```diff
@@ -312,7 +312,7 @@ static void remove_usergroup (void)
 		return;
 	}
 
-	if (!fflg) {
+	if (fflg < 1) {
 		/*
 		 * Scan the passwd file to check if this group is still
 		 * used as a primary group.
```

Comment:
Hmm, this is not covered by the wording I used in the manual page.  I'll try to come up with some better wording.

If anyone has better ideas for the wording, please let me know.